backend/src/main/java/com/webappsbusters/packagemanagement/service/ParcelService.java

```diff
@@ -0,0 +1,25 @@
+package com.webappsbusters.packagemanagement.service;
+
+import com.webappsbusters.packagemanagement.domain.Parcel;
+import com.webappsbusters.packagemanagement.repository.ParcelRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+@Service
+@Transactional
+public class ParcelService {
```

Comment:
I think we should have interface and implementation, but it can stay as it is now